superset-frontend/cypress-base/cypress/support/e2e.ts

```diff
@@ -18,9 +18,32 @@
  */
 import '@cypress/code-coverage/support';
 import '@applitools/eyes-cypress/commands';
+import failOnConsoleError, { Config } from 'cypress-fail-on-console-error';
 
 require('cy-verify-downloads').addCustomCommand();
 
+// fail on console error, allow config to override individual tests
+// these exceptions are a little pile of tech debt
+const { getConfig, setConfig } = failOnConsoleError({
+  consoleMessages: [
+    /\[webpack-dev-server\]/,
+    'The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".',
+    'The pseudo class ":nth-child" is potentially unsafe when doing server-side rendering. Try changing it to ":nth-of-type".',
+    /Warning: /,
```

Comment:
Is it possible to make this regex more restrictive and add specific warning types? My concern is that any new warning will be ignored.

Reply:
Yes... that was going to be my next PR. Just trying to get the framework through. I can add it here, though.